pointerevents/pointerevent_constructor.html

```diff
@@ -30,6 +30,8 @@
             var testHeight = 5;
             var testTiltX = -45;
             var testTiltY = 30;
+            var testButton = 0;
+            var testButtons = 1;
```

Comment:
The [PR I've opened](https://github.com/jquery/PEP/pull/313/commits/f0970758abc7be5817fb1a94ca67afb15a12b924) to support https://github.com/w3c/pointerevents/issues/146 in PEP sets pressure to 0 when buttons is 0.  Thus, if someone tried to create a `PointerEvent` with `pressure` set but not `buttons`, pressure would be 0.

This seems reasonable.    `pressure` without `buttons` would be nonsense, and allowing a constructor implementation to validate those parameters makes it really easy to assert correctness.  (Look at how simple [the PEP PR](https://github.com/jquery/PEP/pull/313/commits/f0970758abc7be5817fb1a94ca67afb15a12b924) is).

If allowing `PointerEvent` arguments to be nonsensical is a requirement, that should be its own test.